android-sdk/src/main/java/org/sagebionetworks/bridge/android/manager/AuthenticationManager.java

```diff
@@ -269,35 +251,26 @@ public UserSessionInfo getUserSessionInfo() {
         // TODO: a way to distinguish if session is null because we haven't signed on, or if it
         // was invalidated
         UserSessionInfo session = apiClientProvider.getUserSessionInfoProvider()
```

Comment:
Will that work? For example, if during a task or something, the consent needs to be retrieved, does there need to be a version that doesn't do a network call and can be run on the main thread? Or can something like AsyncTask take care of that?